resultsdbupdater/consumer.py

```diff
@@ -25,7 +25,7 @@ def consume(self, msg):
         if msg['topic'] == '/topic/VirtualTopic.eng.platformci.tier1.result':
             self.log_msg(msg)
             return ci_metrics_post_to_resultsdb(msg)
-        elif msg['topic'] == '/topic/VirtualTopic.eng.cips':
+        elif msg['topic'] == '/topic/VirtualTopic.eng.cips.complete':
```

Comment:
I think we probably want to store errors as well, not just complete results. @gsr-shanks or @danc86 any opinion?

Reply:
@mprahl I agree, we should store errors as well. Right now I don't have a list of errors. The errors list should be defined in "FACTORY 2.0 CI UMB messages".